packages/shared/src/lib/cores.ts

```diff
@@ -14,3 +14,21 @@ export function hasAccessToCores(user: LoggedUser): boolean {
 
   return !verifyCoresRole(user, CoresRole.None);
 }
+
+export function canReceiveCores(user: LoggedUser): boolean {
+  if (isNullOrUndefined(user?.coresRole)) {
+    return false;
+  }
```

Comment:
I _think_ this can be omitted from this, as `verifyCoresRole` will handle null/undefined.
Had to use it in `hasAccessToCores` because I flip the return

Reply:
Yep :) tested and can be removed